docs/experiments/project-depends-on.md

```diff
@@ -29,7 +35,13 @@ rules:
   languages: [python]
 ```
 
-## Limitations
+## Supported Languages
 
-* Dependency resolution supports **only Yarn, npm, and Pip lockfiles** (pipfile.lock, yarn.lock, package-json.lock)
```

Comment:
I think it might be good to note that we are still only parsing lockfiles, as a broad limitation

Reply:
Maven's `pom.xml` isn't a lockfile, it's a dependency manifest, so this is no longer true

Reply:
Good point! I still think some caveat might be appropriate, as I think users might expect we parse `package.json` for npm -- but we don't. Maybe a "unsupported" column below?

Reply:
Added a section explaining this